internal/services/jwtService.go

```diff
@@ -0,0 +1,23 @@
+package user
+
+import (
+	"crud-golang/internal/models"
+	"github.com/golang-jwt/jwt/v4"
+	"time"
+)
+
+const jwtSecret = "secret"
```

Comment:
Might be a good idea to read this from the env, or use viper for example
Also, how do you feel about having secret per user?